### PR TITLE
More fixes, rephrases, and typos

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

**Comment by navidcy** [comment]
@josuemtzmo have a look!

File: SST/temperature_animation.rst [hunk 1]
```diff
@@ -1,19 +1,15 @@
-=====================
-Temperature animation
-=====================
+SST/temperature_animation.rst=================================
```
Comment (josuemtzmo) [inline SST/temperature_animation.rst]: Can you remove the filename here?

**Review by josuemtzmo — COMMENTED** [review]
It looks great, just change that comment in the header of the` temperature_animation.rst` file.

File: SST/temperature_animation.rst (hunk 1 above)
Comment (navidcy) [inline SST/temperature_animation.rst]:
```suggestion
=================================
```

**Review by josuemtzmo — APPROVED** [review]
Awesome! Thanks @navidcy, it looks great!

**Pull request merged**